libr/bin/format/elf/elf.c

```diff
@@ -2700,12 +2701,12 @@ static size_t get_next_not_analysed_offset(ELFOBJ *bin, size_t section_offset, s
 	return offset;
 }
 
-static size_t populate_relocs_record_from_section(ELFOBJ *bin, RBinElfReloc *relocs, size_t pos) {
-	size_t size, i, j;
+static void populate_relocs_record_from_section(ELFOBJ *bin, RBinElfReloc *relocs, size_t num_relocs, size_t *pos) {
+	size_t size, i, j, p = 0;
```

Comment:
Aren't you resetting the relocs index to zero this way?? This will overwrite the relocations found in the Dynamic segment

Reply:
it was entering the 2nd loop with p > end and caused 0 relocs to be printed

Reply:
this is wrong. this way you just overwrite the relocation entries read from the dynamic segment.